docs/concepts/services-networking/ingress.md

```diff
@@ -60,6 +60,8 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: test-ingress
+  annotations:
+    ingress.kubernetes.io/rewrite-target: /
```

Comment:
I think the point of this doc is to only document the cross platform features and link to the platform specific docs. rewrite-target won't work on GKE/GCE/AWS afaik, so probably doesn't belong here. There's already an example for it in the main repo: https://github.com/kubernetes/ingress/tree/master/examples/rewrite/nginx, maybe it just needs to be surfaced better in the nginx controller README?